### ISSUE TITLE
Lazy-loading plugin if no element defined at all

### ISSUE THREAD

**Issue by r-brown** [Labs64/GuideChimp#60]
**Is your feature request related to a problem? Please describe.**
[Lazy-loading](https://github.com/Labs64/GuideChimp/tree/master/plugins/lazyLoading) plugin should not be effective for one particular step, if no element defined.

**Describe the solution you'd like**

In the below example (with enabled `Lazy-loading` plugin), the first step should wait for an element, and the second not and appear immediately, as there is no `element` configuration defined.

```
{
    "/dashboard/": [
        {
            "element": ".create-survey",
            "title": "Create Survey",
            "description": "Get survey responses from around the world in minutes.<br>Easily find your ideal respondents for market research with SurveyMonkey Audience.",
            "position": "bottom"
        },
        {
            "title": "Improve GuideChimp Public Tours",
            "description": "Support community and help to improve <i>GuideChimp Public Tours</i>!<br>Know how to make this guided tour better or want to enable your tour?<br><br>Contribute to the <a href='https://github.com/Labs64/GuideChimp-tours' target='_blank'>GuideChimp Public Tours repository</a>"
        }
    ]
}
```

**Comment by v-rudkovskiy** [comment]
I've already done it in the previous version, see https://github.com/Labs64/GuideChimp/blob/f9b35af0fcb5d81fe58669863803712379d6ce9d/plugins/lazyLoading/index.js#L27

**Comment by r-brown** [comment]
Fixed in the release [2.4.2](https://github.com/Labs64/GuideChimp/releases/tag/2.4.2)
Dependent [GuideChimp Chrome Extension](https://chrome.google.com/webstore/detail/guidechimp-chrome-extensi/afecedbgkfoijeligfjflidfddndnjng) updated to the above version.
